src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -99,22 +40,13 @@ function GridContainer(props: RelayProps) {
       relay.loadMore(PAGE_SIZE, x => x)
     })
 
-  const handleScroll = ({ nativeEvent }) => {
-    if (isCloseToBottom(nativeEvent)) {
-      loadMore()
-    }
-  }
-
   return (
-    <ScrollView onScroll={handleScroll} scrollEventThrottle={30}>
+    <ScrollView onScroll={isCloseToBottom(loadMore)} scrollEventThrottle={400}>
```

Comment:
When attempting to use a <FlatList /> + end detection in these nested containers it would incorrectly state that we've reached the bottom and constantly refetch. I spent a ton of time playing with it but couldn't figure out why. In the interest of time went with onScroll + new util as seen in various other places in app. (Can return later to figure out)

Reply:
I actually think `<ScrollView />` is more appropriate than `<FlatList />` in this case. Getting `onEndReached` comes at the cost of having to disable scrolling in `<GenericGrid />` and using the API in a hacky way whereas a ScrollView is actually what we want in this case.

Reply:
👍

I see what you mean by having to disable scrolling in the grid component. I’m not sure I know what you mean by using the API in a hack way, which I’d love to hear more about but that’s just out of interest.

Reply:
Right, what I meant by "hacky" is we're essentially making a `FlatList` with one row only to use end detection so it's not really a list, we'd have to create a list with one item just to render our row that contains the grid component. In this case, I don't think what we get from the FlatList API justifies the effort.

Reply:
Ah yes, makes total sense. Originally I hadn’t at first made the connection that there was a single row, I only focused on the discussion that a ListView with data source was cumbersome and that being the reason to ditch it.